src/components/views/emojipicker/EmojiPicker.tsx

```diff
@@ -223,6 +224,10 @@ class EmojiPicker extends React.Component<IProps, IState> {
         }
     };
 
+    private reactWith = (reaction: string) => {
```

Comment:
```suggestion
    private reactWith = (reaction: string): void => {
```

Reply:
Makes sense – should I add it to all the other callbacks while I'm at it?

Reply:
Yeah, sure go for it!

Reply:
Went all out and made Typescript happier all over the place in adbb4a4df8.